File: prak4/PRAK104-2510817210003-MuhammadRaffaAzisha.py
sepatu_a = 400000
diskon_a = 13.0
sepatu_b = 350000
diskon_b = 21.0

hasil_a = sepatu_a - (sepatu_a * (diskon_a / 100))
hasil_b = sepatu_b - (sepatu_b * (diskon_b / 100))

print(f"Harga awal Sepatu A adalah Rp {sepatu_a:.0f}")
print(f"Harga awal Sepatu B adalah Rp {sepatu_b:.0f}")
print(f"Sepatu A mendapat diskon {diskon_a:.0f}% sehingga harganya menjadi Rp {hasil_a:.0f}")
print(f"Sepatu B mendapat diskon {diskon_b:.0f}% sehingga harganya menjadi Rp {hasil_b:.0f}")

File: prak4/PRAK104-2510817210003-MuhammadRaffaAzisha.c
#include <stdio.h>

int main() {
    double SepatuA = 400000;
    double diskonA = 13.0;
    double hasilA;
    double SepatuB = 350000;
    double diskonB = 21.0;
    double hasilB;

    hasilA = SepatuA - (SepatuA * (diskonA / 100.0));
    hasilB = SepatuB - (SepatuB * (diskonB / 100.0));

    printf("Harga awal Sepatu A adalah Rp %.0f\n", SepatuA);
    printf("Harga awal Sepatu B adalah Rp %.0f\n", SepatuB);
     printf("Sepatu A mendapat diskon %.0f%% sehingga harganya menjadi Rp %.0f\n", diskonA, hasilA);
    printf("Sepatu B mendapat diskon %.0f%% sehingga harganya menjadi Rp %.0f\n", diskonB, hasilB);

    return 0;
}